usaspending_api/common/filters/time_period/time_period.py

```diff
@@ -1,28 +1,29 @@
 from abc import ABC, abstractmethod
+from types import list
 
 
-class ITimePeriod(ABC):
+class TimePeriod(ABC):
```

Comment:
I still think there's value in signifying in the name that this isn't a class that can be instantiated directly. So: `AbstractTimePeriod` or `ATimePeriod` (we've only used `Abstract` as a prefix to my knowledge in the past, when adding that kind of detail to the class name).

Reply:
Just made that change